macos/FluentUI/Button.swift

```diff
@@ -0,0 +1,188 @@
+//
+// Copyright Microsoft Corporation
+//
+
+import AppKit
+
+fileprivate struct Constants {
+	
+	static let borderWidth: CGFloat = 1
+	
+	static let cornerRadius: CGFloat = 3
+	
+	static let minimumWidth: CGFloat = 120
```

Comment:
Interesting. Just tested this, and it seems NSButton does have a minimum width of 28.
I added the minimum width originally because it was in a design for another button I made. 

![Screen Shot 2020-05-20 at 2 40 30 PM](https://user-images.githubusercontent.com/6722175/82500535-63eed080-9aa8-11ea-8fdd-ce90a384aa5f.png)

Here's what FluentButton looks like with a minimum width of 30.. I feel I prefer this.

![Screen Shot 2020-05-20 at 2 48 47 PM](https://user-images.githubusercontent.com/6722175/82500870-1de63c80-9aa9-11ea-98f0-1dd59909573e.png)

Reply:
Actually, it looks like NSButton doesn't have a minimum width, just nicely enforced horizontal padding. I removed the minimum width, and increased the horizontal padding to compensate